reana_commons/openapi_specifications/reana_workflow_controller.json

```diff
@@ -457,7 +532,7 @@
             "type": "string"
           },
           {
-            "description": "Optional. Extra parameters for workflow status.",
+            "description": "Optional. Operational and input parameters for workflow execution.",
```

Comment:
"Input and operational..." to keep alphabetical order, which nicely corresponds to the order of "scientific significance" as well!

Perhaps "Additional input and operational..." to better explain the optional nature of the thing...